src/app/_components/action-snack-bar-with-modal/snack-bar-component-example-snack.html

```diff
@@ -0,0 +1,9 @@
+<span>
+  Hello! Please set your display name. This is the name that will be shown when you post, comment, etc on the site, and will allow users to recognize who you are.
+</span>
+<button mat-raised-button
+  (click)="openDialog()"> Add Display Name </button>
+
+<!-- Copyright 2018 Google Inc. All Rights Reserved.
+    Use of this source code is governed by an MIT-style license that
+    can be found in the LICENSE file at http://angular.io/license -->
```

Comment:
@chenaicirt is there a reason there is license text in this file in particular?

Reply:
Nope, this was just taken from the snackbar demo code that was on the material website

Reply:
should I delete this?